lib/features/feed/widgets/share_sheet.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:fluttercon/common/data/enums/social_platform.dart';
+import 'package:fluttercon/common/data/models/local/local_feed.dart';
+import 'package:fluttercon/common/utils/constants/app_assets.dart';
+import 'package:fluttercon/common/utils/misc.dart';
+import 'package:fluttercon/core/theme/theme_colors.dart';
+import 'package:fluttercon/features/feed/cubit/share_feed_post_cubit.dart';
+import 'package:fluttercon/features/feed/widgets/social_media_button.dart';
+import 'package:fluttercon/l10n/l10n.dart';
+
+class ShareSheet extends StatelessWidget {
+  const ShareSheet({
+    required this.feed,
+    required this.l10n,
+    super.key,
+  });
+
+  final LocalFeedEntry feed;
+  final AppLocalizations l10n;
```

Comment:
This doesn't need to be passed from the parent. It can be initialised in the build method.